_episodes_rmd/03-r-data-analysis.Rmd

```diff
@@ -33,258 +32,315 @@ knitr_fig_path("03-")
+In this case, it says that that we used `=` instead of `==`. 
+That's because we use `=` (single equals) when naming arguments that you are passing to functions.
+So here R thinks we're trying to assign 1990 to year, kind of like we do in the summarize function.
+What we really want to do is find all of the years that are equal to `r smoking %>% summarize(earliest_year=min(year)) %>% pull(earliest_year)`. 
+To do this, we have to use `==` (double equals), which we use when testing if two values are equal to each other:
+
+```{r, filtSummarize}
+smoking %>%
+  filter(year == 1990) %>%
+  summarize(mean_smoke_pct_1990=mean(smoke_pct))
 ```
 
+Notice how the pipe operator (`%>%`) allows us to combine these two simple steps into a more complicated data extraction? We took the data, filtered out the rows, then took the mean value. The argument we pass to `filter()` needs to be some expression that will return TRUE or FALSE. We can use comparisons like `>` (greater than) and `<` (less than) for example. Here we tested for equality using a double equals sign `==`. 
+
 > ## Filtering the dataset
 >
-> What is the average GDP per capita for the first year in the dataset? *Hint: the column headers identified by `read_csv()` showed us there was a column called gdpPercap in the dataset*
+> What is the average percent of people with lung cancer for the most recent year in the dataset? *Hint: the column headers identified by `read_csv()` showed us there was a column called `lung_cancer_pct` in the dataset*
+>
+> Bonus: Figure out a way to write the code so you don't have to type the year into filter. 
+> (Typing in the year is an example of hard-coding and we try to avoid it because if the data changes the number might also change.)
 >
 > > ## Solution
 > >
-> > Identify the earliest year in our dataset using `min()` and `summarize()`
+> > Identify the earliest year in our dataset using `max()` and `summarize()`:
 > >
 > > ```{r FilterChallenge1}
-> > gapminder_data %>%
-> > summarize(first_year=min(year))
+> > smoking %>%
+> > summarize(recent_year=max(year))
 > > ```
 > >
-> > We see here that the first year in the dataset is 1952. Filter to only 1952, and determin the average GDP per capita.
+> > Filter to only the maximum year, and determine the average percent of people with lung cancer.
 > >
 > > ```{r FilterChallenge2}
-> > gapminder_data %>%
-> > filter(year == 1952) %>%
-> > summarize(average_gdp=mean(gdpPercap))
+> > smoking %>%
+> > filter(year == 2019) %>%
+> > summarize(mean_cancer_pct=mean(lung_cancer_pct))
 > > ```
 > > {: .source}
-> > By combining `filter()` and `summarize()` we were able to calculate the mean GDP per capita in the year 1952.
+> > By combining `filter()` and `summarize()` we were able to calculate the mean mercent of people with lung cancer in 2019
+> > 
+> > Bonus:
+> > Instead of finding the minimum year first and then filtering, we can find the minimum year right in the filter function: 
```

Comment:
```suggestion
> > Instead of finding the most recent year first and then filtering, we can find the most recent year right in the filter function: 
```